solidity/test/integration/LlamaVesting.t.sol

```diff
@@ -17,47 +17,77 @@ contract IntegrationLlamaVesting is IntegrationBase {
     _vestingStartTime = _connextVestingWallet.start();
   }
 
-  function test_VestAndUnlock() public {
-    vm.prank(payer);
-    _llamaPay.depositAndCreate(TOTAL_AMOUNT, address(_connextVestingWallet), PAY_PER_SECOND);
-
+  function test_VestAndUnlock_LaunchDate() public {
     // Before the cliff
-    uint256 _timestamp = _connextVestingWallet.cliff() - 1;
-    uint256 _vestedAmount = (_timestamp - _vestingStartTime) * PAY_PER_SECOND / 1e2;
-
+    uint256 _timestamp = LAUNCH_DATE;
     // The unlocking contract holds the tokens
     _warpAndWithdraw(_timestamp);
-    _assertBalances(0);
-    assertEq(_nextToken.balanceOf(address(_connextVestingWallet)), _vestedAmount);
+    _assertWalletBalance(6_838_356 ether);
+    _assertOwnerBalance(0 ether);
+  }
+
+  function test_VestAndUnlock_1SecBeforeCliff() public {
+    // 1 secont before the cliff
+    uint256 _timestamp = LAUNCH_DATE + 365 days - 1;
```

Comment:
use `SEP_05_2022` style in favour of `LAUNCH_DATE`
i prefer `SEP_05_2022 + 2 * YEAR + YEAR / 12` style 👍